src/components/AppHeader/AppHeader.tsx

```diff
@@ -36,17 +55,47 @@ const AppHeader: FC<HeaderProps> = () => {
       >
         <Flex $justifyContent={"center"} $alignItems={"center"}>
           <OakLink page={"home"}>
-            <Logo height={48} width={104} />
+            <MobileWrapper>
```

Comment:
Could you use a Box with display={["flex","none"]} here rather than making these new wrappers?